db/schema.rb

```diff
@@ -1439,9 +1439,9 @@
 
   create_table "users_gdpr_delete_requests", force: :cascade do |t|
     t.datetime "created_at", precision: 6, null: false
-    t.string "email"
+    t.string "email", null: false
     t.datetime "updated_at", precision: 6, null: false
-    t.integer "user_id"
+    t.integer "user_id", null: false
```

Comment:
Can you exclude the changes to this file from the PR? It looks like they got committed accidentally.

We've been getting a lot of PRs with these lines changed in them, so I checked and DEV's production DB _does_ have these constraints, so it was probably a mistake that these didn't get committed before, but we'll fix it separately.

Reply:
Yeah, I'll do that. The changes were unintentional but I wasn't sure if I should revert them so pushed it anyway.

Reply:
All good. It's probably our fault. :joy: